inference-engine/tests/functional/plugin/myriad/subgraph_tests/dsr_tests_common.hpp

```diff
@@ -30,14 +30,17 @@ class DSR_TestsCommon : virtual public LayerTestsUtils::LayerTestsCommon {
     std::unordered_map<std::string, DataShape> m_shapes;
     ngraph::ParameterVector m_parameterVector;
 
+    std::shared_ptr<ngraph::Function> m_testFunction;
+    std::shared_ptr<ngraph::Function> m_refFunction;
```

Comment:
What is the difference between `function`, `m_testFunction` and `m_refFunction`? Why you need all 3 of them?

Reply:
Now I see there is no real need to split functions. Fixed